Tools/ssl/multissltests.py

```diff
@@ -102,6 +108,12 @@
         "OpenSSL and LibreSSL versions are given."
     ).format(LIBRESSL_RECENT_VERSIONS, LIBRESSL_OLD_VERSIONS)
 )
+parser.add_argument(
+    '--awslc',
+    nargs='+',
+    default=(),
+    help=f"AWS-LC versions, defaults to '{AWSLC_RECENT_VERSIONS}'."
```

Comment:
Considering the line 21 compatibility request and the format of other arguments:

```suggestion
    help=(
        "AWS-LC versions, defaults to '{}' if no crypto library versions are given."
    ).format(AWSLC_RECENT_VERSIONS)
```